### DISCUSSION TITLE
Create note from external application on Android

### DISCUSSION THREAD

**Discussion by SophieLabs** [standardnotes/forum#2500] [Suggestions]
Is it possible to create notes from an external application? That is, with other note taking apps, if I'm browsing the web and want to make a note of the page, I can hit the share button and select the note taking app, and it will create a note with the link.

From what I can see, this is not possible in Standard Notes. Instead, I would have to copy the address, switch to the app and create a note manually. Not a huge deal, but the share method has a much nicer flow to it.

**Comment by atmoio** [comment]
This isn't currently possible, but definitely in the realm of reasonable functionality. I hope to add this to a future release soon.

**Comment by scolestock** [comment]
(Would love to see this in iOS share options as well)

**Comment by MorgothSauron** [comment]
+1 for Standard Notes available in Android "share menu"

**Comment by zngr** [comment]
+1 since I really miss this feature ;)
Any news regarding the timeline?

**Comment by atmoio** [comment]
We have an outstanding pull request for this feature. Still need to review it.

**Comment by g0hz** [comment]
+1 for iOS & Android share/actions or some doc on URL Scheme etc..

**Comment by jetsem** [comment]
+1 for iOS & Android share/actions or some doc on URL Scheme etc..

**Comment by vince-p** [comment]
Is there any update on this? This is the key missing feature that is stopping me from subscribing

**Comment by JaspalSuri** [comment]
We hope to add this feature before or around v4, our next long-term update. 🙂
